src/components/ui/Link/Link.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import type { LinkProps } from '.'
+import Link from '.'
+
+export default {
+  component: Link,
+  title: 'Atoms/Link',
+  argTypes: {
+    to: {
+      type: { name: 'string', required: true },
+    },
+    ref: {
+      table: {
+        disable: true,
+      },
+    },
+    as: {
+      table: {
+        disable: true,
+      },
+    },
+    inverse: {
+      control: 'boolean',
+      table: { defaultValue: false },
+    },
+  },
+}
+
+const Template = ({ inverse, ...args }: LinkProps) =>
```

Comment:
@lucasfp13 what do you think?

https://github.com/vtex-sites/gatsby.store/pull/19#issuecomment-1116221630

Reply:
awesome! @hellofanny 💯